### PR TITLE
Add getversion

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
Needed for https://github.com/conda-forge/staged-recipes/pull/10619

File: recipes/getversion/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+    - six
```
Comment (xhochy) [inline recipes/getversion/meta.yaml]: Shouldn't these all below go into run?

Comment (dpryan79) [reply recipes/getversion/meta.yaml]: Maybe? So far I've only seen them in `setup.py` rather than the actual python module.

Comment (dpryan79) [reply recipes/getversion/meta.yaml]: The package is a mess :(

Comment (npavlovikj) [reply recipes/getversion/meta.yaml]: I was playing with this package locally, and those dependencies do need to be in `host`. And I second Devon that the package is a mess :/

**Pull request closed without merging**